### PR TITLE
feat(tokens): add new tokens, update app background for system

### PR DESCRIPTION
**Pull request by williamernest**
Adds the new tokens for the system. Exports typography tokens as an alias and a global (exact mapping, will remove global in the future). Only change to applications is a minor shift in color for the --cds-alias-object-app-background color for accessibility contrast ratio reasons.

### CODE REVIEW

**Review by kevinbuhmann — APPROVED** [review]

**Review by kevinbuhmann — APPROVED** [review]
I am assuming all of the colors are correct. Have a designer review if needed.

**Comment by astorije** [comment]
Hi there! Are all these new tokens going to be added to [Storybook](https://storybook.core.clarity.design/?path=/story/foundation-design-tokens--page)?

**Comment by williamernest** [comment]
@astorije Yes, they will be documented in the future. We're focusing on Angular v16 at the moment, but we will circle back after it is released.

File: projects/core/build/tokens.ts [hunk 1]
```diff
@@ -536,7 +561,7 @@ const aliases = {
       },
     },
     app: {
-      background: token(color.gray[50]),
+      background: token(color.construction[25]),
```
Comment (astorije) [inline projects/core/build/tokens.ts:564]: Just checking: this effectively changes the background of our applications from #fafafa to #fcfdfd, is that intentional?

**Comment by astorije** [comment]
@williamernest, noted, thanks. We'll probably hold off on upgrading our codebase to v6.6.0 for now then, until it is all squared away.

File: projects/core/build/tokens.ts (hunk 1 above)
Comment (williamernest) [reply projects/core/build/tokens.ts:564]: Yes

**Pull request merged**